### PR TITLE
LPS-193770 - Create confirm object unbinding modal

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPS-193770

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ModalUnbindObject.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+	openToast,
+} from '@liferay/object-js-components-web';
+import {createResourceURL, fetch, sub} from 'frontend-js-web';
+import React, {useEffect, useState} from 'react';
+
+import './ModalUnbindObject.scss';
+
+interface ModalUnbindObjectProps {
+	baseResourceURL: string;
+	onvisibilityChange: () => void;
```
Comment (naraandrade) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ModalUnbindObject.tsx:21]: @danvb112 Same comment as in the other PR about capitalizing the 'v'.

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Review by naraandrade — APPROVED** [review]

**Comment by MarinhoFeliphe** [comment]
Resent here: https://github.com/liferay-objects/liferay-portal/pull/2508

**Pull request closed without merging**